src/pages/ViewScholarship.jsx

```diff
@@ -47,6 +49,30 @@ export default function ViewScholarship({ history, location, match }) {
       <Helmet>
         <title>{scholarship.data.name}</title>
       </Helmet>
+
+      {location?.state?.status && (
+        <Alert
+          color="primary"
+          variant="filled"
+          action={
+            <>
+              <IconButton
+                size="medium"
+                color="inherit"
+                onClick={() =>
+                  history.replace(`/scholarships/${scholarship.id}`)
+                }>
+                <CloseIcon fontSize="small" />
+              </IconButton>
+            </>
+          }>
+          <AlertTitle sx={{ color: '#fff' }}>Success</AlertTitle>
+          {`Scholarship successfully ${
+            location?.state?.status === 'add' ? 'submitted.' : 'updated.'
```

Comment:
You can use `scholarship` itself. 
```suggestion
            scholarship.data.dateAdded === scholarship.data.lastModified ? 'submitted.' : 'updated.'
```

Reply:
had to use `getTime()` to compare